test/xbgpu/test_engine.py

```diff
@@ -292,28 +292,24 @@ def verify_corrprod_data(
     corrprod_results
         List of arrays of all GPU-generated data from
         :class:`TestEngine._send_data`.
-    batch_indices
-        Batch indices used to generate stimulus data for the XBEngine. See
-        :class:`TestEngine.test_engine_end_to_end` for more details.
+    acc_indices
+        Accumulation indices used to generate stimulus data for the XBEngine.
```

Comment:
Hmmmm, maybe this description requires an update. Let me know if you're of the same opinion.

Reply:
I agree - my first reaction on seeing the type annotation was "why a list of lists"? Maybe add something like "This is a list of lists, with the outer index matching `corrprod_outputs`."